### ISSUE TITLE
reproduce issue

### ISSUE THREAD

**Issue by xxyzll** [TencentARC/GRPO-CARE#1]
Hi, thank you very much for your valuable work—it’s taught me a lot. However, I ran into a bit of trouble when trying to reproduce it.

I used 8×A100 GPUs to run run_grpo_care_margin0.01_video_r1.sh. Since you didn’t provide detailed implementation instructions, I inferred from the code that the model used is Qwen-2.5-vl-7B (without SFT). So, I downloaded and ran the code from the official repository without modifying any other details.

However, I encountered a few issues (on the VideoMMMU benchmark):

1. The model seems to have lost its reasoning ability. For example:
`{ "problem_id": 1, "problem": "What kind of sugar is this, aldose or ketose?", "data_type": "video", "problem_type": "multiple choice", "original_question_type": "multiple-choice", "options": [ "A. aldose", "B. ketose", "C. glucose", "D. fructose", "E. ribose", "F. xylose", "G. arabinose", "H. mannose", "I. galactose", "J. sucrose" ], "process": "<think>A</think>", "solution": "<answer>A</answer>", "path": "./Evaluation/VideoMMMU/dev_Biology_3.mp4", "data_source": "https://www.youtube.com/watch?v=U7YqCUd_ZfA", "output": "<think>A</think>\n\n<answer>A</answer>", "prediction": "A", "reward": 1.0, "correct": true }`
In the <think></think> tag, there is no reasoning—just a repetition of the final answer.

2. The model's performance is underwhelming. On VideoMMMU, I only achieved: 47.4 (FPS 16). This result is noticeably lower than what was reported in the paper. Even after increasing the FPS to 32, the performance did not significantly improve (48.6).

**Comment by ChenYi99** [comment]
Could you provide more details about your implementation?

**Comment by xxyzll** [comment]
I downloaded the Qwen2.5-vl-7B-instruct base model from the official ModelScope repository (Qwen). Then, I ran the script you provided: run_grpo_care_margin0.01_video_r1.sh. Other than increasing the number of GPUs from 4 to 8, I didn’t change any settings.

**Comment by ChenYi99** [comment]
Did you use the full training dataset from Video-R1, or just a subset? Also, please check the debug_log file for more details about the issue. You could try training with 4 GPUs for 1000 steps to see if the problem persists,  although when we tested with 8 GPUs, we didn’t encounter this issue. Additionally, you can check the model’s performance on other evaluation datasets as well.

**Comment by xxyzll** [comment]
1. I downloaded the complete dataset and believe the data is correct. When I trained Video-R1 using Video-R1-260k, I was able to obtain the correct results.
2. I noticed that you skipped SFT fine-tuning and went straight to reinforcement learning. During execution, I observed that the format wasn’t entirely correct. With further fine-tuning, I also found that the format reward was decreasing.

![Image](https://github.com/user-attachments/assets/a7709153-ba49-46ac-a541-82ea80602989)

**Comment by ChenYi99** [comment]
Thank you for your feedback. Our results indicate the reward curve is stable. We kindly suggest double-checking your setup to identify any potential issues.
